packages/frontend/src/components/ProjectConnection/UpdateProjectConnection.tsx

```diff
@@ -90,7 +93,7 @@ const UpdateProjectConnection: FC<{
             await mutateAsync({
                 name,
                 dbtConnection,
-                warehouseConnection: warehouseConnection!,
+                warehouseConnection: warehouseConnection,
```

Comment:
the original code mistakenly used `!` - the warehouseConnection is always defined here